packages/yaml/test/yaml-document.test.ts

```diff
@@ -251,6 +251,26 @@ seq1:
         });
     });
 
+    describe('deleteAt', () => {
+        it('delete node in existing empty sequence', async () => {
```

Comment:
Could you please add tests to cover the cases where the node does not exist and when the node exists but is not a sequence?

Reply:
8c7e5b3